.github/workflows/added_posts.yml

```diff
@@ -41,6 +41,7 @@ jobs:
           for added_file in ${added_files[@]}; do
             if [[ $added_file == content/notices/* ]]; then
               echo "File added: ${added_file}."
+              sleep 120
```

Comment:
Dirty hack to prevent the notification race condition where alerts go before the blog is live.